packages/client/components/ActionSidebarAgendaItemsSection.tsx

```diff
@@ -26,9 +26,19 @@ const ActionSidebarAgendaItemsSection = (props: Props) => {
     (phase) => phase.phaseType === NewMeetingPhaseTypeEnum.updates
   )!
   const isUpdatesNavigable = updatesPhase && updatesPhase.stages![0].isNavigable
+  const agendaItemsPhase = meeting.phases!.find(
+    (phase) => phase.phaseType === NewMeetingPhaseTypeEnum.agendaitems
+  )!
+  const agendaItems = useMemo(() => {
```

Comment:
this component isn't directly using `agendaItems`, it's only passing it down along with the `meeting` parent object. may be cleaner to just pass down `meeting` and let the child component get what it needs